docker/common.sh

```diff
@@ -42,6 +43,7 @@ if_centos install_packages \
     pkgconfig
 
 if_ubuntu install_packages \
+    adduser \
```

Comment:
why was this added?

Reply:
Both bzip2 and adduser were necessary during the build. I'm rerunning it now to give a more detailed answer

Reply:
Both of them are due to crosstool-ng.sh, so I'm moving the installations there, and documenting the reasons in the commit message